pom.xml

```diff
@@ -227,7 +227,7 @@
     <version.org.kie.workbench.app>${version.org.kie}</version.org.kie.workbench.app>
 
     <version.org.wildfly.swarm>2018.3.3</version.org.wildfly.swarm>
-    <version.net.byte-buddy>1.4.16</version.net.byte-buddy>
+    <version.net.byte-buddy>1.8.17</version.net.byte-buddy>
```

Comment:
do we know why we manage byte-buddy?
just got from another change that it is used in mockito and optaweb-employee-rostering override managed version due this https://github.com/kiegroup/optaweb-employee-rostering/blob/master/pom.xml#L86

Reply:
I can't find a dependency of org.mocikito:mockito-core: [1.1.0.19](https://search.maven.org/artifact/org.mockito/mockito-core/1.10.19/jar) to net.byte-buddy

Reply:
I am talking about 2.x look at https://repository.jboss.org/nexus/index.html#nexus-search;gav~org.mockito~mockito-core~~~~kw,versionexpand%20
That is used in https://github.com/kiegroup/optaweb-employee-rostering/blob/master/pom.xml